### PR TITLE
Schema issue template

### PR DESCRIPTION
**Pull request by hmlanigan**
## Description

Add an issues template for schema changes. Give contributors an understanding of what we're looking for.

## QA steps

No complicated QA is needed, maybe wording could be checked. Also in the Files Changed section, there's a View File button in the top right ... menu for individual files, which is useful to see what the Gİthub form will look like.

### CODE REVIEW

**Comment by amandahla** [comment]
What do you think about adding a field/checked box/label (?)  to make it clear that changes an existing one? Since this could have more impact/breaking changes than adding a new one.

File: .github/ISSUE_TEMPLATE/SchemaChanges.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        },
+    validations:
+      required: true
+
+  - type: textarea
+    id: Info
+    attributes:
+      label: "Notes & References"
+      description: |
+        Please add relevant notes, links to mattermost chats, other related
```
Comment (cderici) [inline .github/ISSUE_TEMPLATE/SchemaChanges.yml:41]: "........ links to matrix (or element) chats.........."

File: .github/ISSUE_TEMPLATE/SchemaChanges.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      placeholder: ex. "I plan to implement data source resource X in the provider."
+      render: terraform
+    validations:
+      required: true
+
+  - type: textarea
+    id: Details
+    attributes:
+      label: Details
+      description: Details about each schema attribute to be added or modified including structure contents. Include also why choices were made.
```
Comment (cderici) [inline .github/ISSUE_TEMPLATE/SchemaChanges.yml:26]: I'd also add "......... Include also why choices were made, especially about fields being *Optional*, *Computed*, etc."

**Review by cderici — APPROVED** [review]
LGTM % comments

File: .github/ISSUE_TEMPLATE/SchemaChanges.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
+name: Schema Change
+description: You want to contribute functionality to Terraform Juju Provider, requiring a schema change? Proceed with this one.
+labels: [schema changes]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Contributions are happily accepted. This will us understand the change coming and
```
[the remark this answers is not in the document]
Comment (hmlanigan) [reply .github/ISSUE_TEMPLATE/SchemaChanges.yml:8]: done

**Pull request merged**